lib/configure.js

```diff
@@ -456,8 +516,10 @@ module.exports = function() {
       'You\'re ready to auto-deploy using Travis!  Just commit the changes\n' +
       'in .travis.yml and push the commit to the origin/master branch:\n' +
       '\n' +
-      'git commit -m"configure Travis to auto-deploy to GitHub Pages" .travis.yml\n' +
-      'git push origin master\n' +
+      chalk.bold.gray(
```

Comment:
This is with `chalk.bold.italic.bgBlack`:
![integrate](https://cloud.githubusercontent.com/assets/1616846/11071605/8966815a-87d9-11e5-9aaf-bb9894c54916.png)